scripts/completion.sh

```diff
@@ -0,0 +1,72 @@
+# bash completion support for xkbcli.
+
+___xkbcli_main()
+{
+    _init_completion -s || return
```

Comment:
Can you add a comment (maybe at the top of the file) that this comes from the [bash-completion](https://github.com/scop/bash-completion) package? It took me a bit of effort to find it, turns out it's deprecated in the yet-unreleased version 2.12 (though without emitting a deprecation messages).

The function [doc](https://github.com/scop/bash-completion/blob/fdf4456186eb4548ef628e65fb1be73d8e4695e9/bash_completion.d/000_bash_completion_compat.bash#L205) says "Before calling this function, make sure cur, prev, words, and cword are local, ditto split if you use -s.", but it's not done here. Is this a problem?